app/views/results/show.html.slim

```diff
@@ -1,3 +1,4 @@
+- end_of_journey = FeatureFlags.enabled?(:end_of_journey, @check.session_data)
```

Comment:
The logic on this page gets quite complicated, so to keep it clear we want to keep our variables clearly named. An `end_of_journey` boolean suggests it represents whether this is the end of the journey. But this variable actually represents whether the flag is enabled. Can we rename to `end_of_journey_flag` or similar?